core/src/main/scala/com/cloudera/spark/CGroupsMemoryHandle.scala

```diff
@@ -0,0 +1,37 @@
+package com.cloudera.spark
+
+import scala.io.Source
+
+class CGroupsMemoryHandle(cgroupMemStats: Seq[(String, Long)]) extends MemoryGetter {
+
+  override val namesAndReporting: Seq[(String, PeakReporting)] = cgroupMemStats.map(x => {
```

Comment:
nit: I'd follow the spark style convention in this repo too.  Anytime you're passing a lambda with a `=>` in it, use curly braces and no parens.  So

```scala
.map { x =>
 (x._1, IncerementBytes)
}
```

Reply:
Fixed in the latest commit